ibc-apps/ics20-transfer/src/handler/send_transfer.rs

```diff
@@ -54,7 +54,13 @@ where
     let seq_send_path_on_a = SeqSendPath::new(&msg.port_id_on_a, &msg.chan_id_on_a);
     let sequence = send_packet_ctx_a.get_next_sequence_send(&seq_send_path_on_a)?;
 
-    let token = &msg.packet_data.token;
+    // let token = &msg.packet_data.token;
```

Comment:
Let's remove this commented line.

Reply:
Actually, we won't be able to accept this PR. See this [comment](https://github.com/cosmos/ibc-rs/issues/1022#issuecomment-1875499381) plz.